telegram/_payment/stars.py

```diff
@@ -272,6 +373,64 @@ def de_json(
         return super().de_json(data=data, bot=bot)
 
 
+class TransactionPartnerAffiliateProgram(TransactionPartner):
+    """Describes the affiliate program that issued the affiliate commission received via this
+    transaction.
+
+    This object is comparable in terms of equality. Two objects of this class are considered equal,
+    if their :attr:`commission_per_mille` are equal.
+
+    .. versionadded:: NEXT.VERSION
+
+    Args:
+        sponsor_user (:class:`telegram.User`, optional): Information about the bot that sponsored
+            the affiliate program
+        commission_per_mille (:obj:`int`): The number of Telegram Stars received by the bot for
```

Comment:
it's strange that the telegram docs started listing optional arguments first and then required args 🤔